packages/db/prisma/schema.prisma

```diff
@@ -37,7 +37,7 @@ model Activity {
   name      String
   day       DateTime @db.Date
   startTime DateTime @db.Time()
-  endTime   DateTime @db.Time()
+  duration  Int
```

Comment:
can you make this name `durationMinutes` so that it is clear what the units are? make sure to delete the old db migration before creating a new one